packages/components/modules/activity-log/ActivityLogComponent/index.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+
+import { FC, useState } from 'react'
+
+import { Box } from '@mui/material'
+
+import { ActivityLogsFragment$key } from '../../../__generated__/ActivityLogsFragment.graphql'
+import ActivityUserSearchBar from '../ActivityUserSearchBar'
+import DateRangeChip from '../DateRangeChip'
+import EventFilterChip from '../EventFilterChip'
+import LogGroups from '../LogGroups'
+import { useActivityLogs } from '../graphql/queries/ActivityLogsFragment'
+import { ActivityLogComponentProps } from './types'
+
+const ActivityLogComponent: FC<ActivityLogComponentProps> = ({ queryRef }) => {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [dateRange, setDateRange] = useState<[Date | null, Date | null]>([null, null])
```

Comment:
it would be cool if we could use `useForm` as state manager. Or at least use `zod` for schema validation. How are we validating the input?